File: sqlUpdate.py
import MySQLdb
 
db = MySQLdb.connect(host="localhost",user="user",passwd="password",db="school"
 
# creating cursor
cur = db.cursor(MySQLdb.cursors.DictCursor);
 
name= 'BIG BRAIN'
age= 17
gradeLevel= 11
 
# create table as per requirement
sql = "UPDATE students SET gradeLevel = '100' WHERE name = 'BIG BRAIN' "
cur.execute(sql)
 
# disconnecting from server
cur.close()
db.close()

File: sqlUpdate.php
<?php


$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";

// Create connection to the database and server
$conn = new mysqli($servername, $username, $password, $dbname);

// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

//arrange names alphabetically
$sql = "UPDATE students SET gradeLevel = '100' WHERE name = 'BIG BRAIN' ";
$result = $conn->query($sql);


if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
}
else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>

